mscore/inspector/inspector.cpp

```diff
@@ -939,26 +940,15 @@ InspectorSlur::InspectorSlur(QWidget* parent)
 
       Element* e = inspector->element();
       bool sameType = true;
-      Element::Type subtype = Element::Type::INVALID;
 
-      if (e->type() == Element::Type::SLUR_SEGMENT)
-            subtype = toSlurSegment(e)->spanner()->type();
       for (const auto& ee : inspector->el()) {
-            if (ee->type() != Element::Type::SLUR_SEGMENT) {
-                  sameType = false;
-                  break;
-                  }
-            if (toSlurSegment(ee)->spanner()->type() != subtype) {
+            if (ee->accessibleInfo() != e->accessibleInfo()) {
```

Comment:
Well, it cmpares the (translated) strings for Tie and Slur, and uses it later. Don't see where that might break?